recipes/datatable/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "datatable" %}
+{% set version = "0.11.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datatable-{{ version }}.tar.gz
+  sha256: 19c602711e00f72e9ae296d8fa742d46da037c2d3a2d254bdf68f817a8da76bb
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
+
+# numpy and pandas are _optional_ dependencies of dataframe
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+  run:
+    - python
+    # fix overlinking error
+    - libgcc   # [linux]
+    - libcxx   # [osx]
+
+test:
+  imports:
+    - datatable
+  files:
+    - setup.cfg
+  source_files:
+    - src
+    - tests
+    - tests_random
+    - LICENSE
```

Comment:
I guess this isn't actually needed for the tests:
```suggestion
```

Reply:
If you look at https://github.com/h2oai/datatable/blob/v0.11.0/tests/fread/test-fread-api.py#L101 you can see that this is in fact required for the tests.